Description: With usrmerge configured, ant may be called via /bin/ant.
 In this case it fails with:
  /bin/ant: 1: cd: can't cd to /bin/../share/ant/bin/..
 Using 'readlink -f' instead of this cumbersome while loop solves the problem.
Author: Gilles Filippini <pini@debian.org>
Forwarded: no
--- a/src/script/ant
+++ b/src/script/ant
@@ -144,15 +144,7 @@
   progname=`basename "$0"`
 
   # need this for relative symlinks
-  while [ -h "$PRG" ]; do
-    ls=`ls -ld "$PRG"`
-    link=`expr "$ls" : '.*-> \(.*\)$'`
-    if expr "$link" : '/.*' > /dev/null; then
-      PRG="$link"
-    else
-      PRG=`dirname "$PRG"`"/$link"
-    fi
-  done
+  PRG=`readlink -f $PRG`
 
   ANT_HOME=`dirname "$PRG"`/..
 
